lldb/include/lldb/Host/Host.h

```diff
@@ -87,8 +87,15 @@ class Host {
   StartMonitoringChildProcess(const MonitorChildProcessCallback &callback,
                               lldb::pid_t pid);
 
+  /// System log level.
+  enum SystemLogLevel {
+    eSystemLogInfo,
+    eSystemLogWarning,
```

Comment:
Is there a good reason to keep this separate from DiagnosticSeverity?
https://lldb.llvm.org/cpp_reference/namespacelldb__private.html#a3b3fff978827f40d27f288b4f1f29369
Maybe we could factor this into private enums?

Reply:
Good point! https://github.com/llvm/llvm-project/pull/90917